src/components/Footer/Footer.jsx

```diff
@@ -1,7 +1,138 @@
 import React from 'react';
+import styled from 'styled-components';
+import SOCIAL_LIST from './socialData';
+import INFO_LIST from './infoData';
 
-function Footer(props) {
-  return <div>Footer</div>;
+function Footer() {
+  return (
+    <FooterContainer>
+      <ContentContainer>
+        <Intro>
+          <Title>IOI</Title>
+          <Text>
+            IOI는 모든 사람이 사랑하는 일을 하며 살 수 있도록 세상을 바꾸고자
+            합니다.
+          </Text>
+          <Text>KR 한국어</Text>
+        </Intro>
+        <Content>
+          <Title>About</Title>
+          <Text>홈</Text>
+          <Text>기업교육</Text>
+          <Text>채용</Text>
+          <Text>도움말</Text>
+        </Content>
+        <Content>
+          <Title>크리에이터</Title>
+          <Text>크리에이터 센터</Text>
+          <Text>정규 클래스 지원하기</Text>
+          <Text>도움말</Text>
+        </Content>
+        <Content>
+          <Title>고객센터</Title>
+          <CallBtn>문의하기</CallBtn>
+          <Text>오전 10시 ~ 오후 6시 (주말, 공휴일 제외)</Text>
+        </Content>
+      </ContentContainer>
+      <ContentContainer>
+        <SocialList>
+          {SOCIAL_LIST.map(social => (
+            <WrapText key={social.id}>
+              <TextSmall>{social.title}</TextSmall>
+              <MidDot>&middot;</MidDot>
```

Comment:
```suggestion
              <span>&middot;</span>
```
아래 리뷰드린 내용과 연관지어서 변경하자면 이런식으로 적용될 수 있겠습니다.